### DISCUSSION TITLE
How to add a keybinding with this eidtor

### DISCUSSION THREAD

**Discussion by IWANABETHATGUY** [facebook/lexical#1805] [Q&A] [answered]
For example, i want to add a shortcut  when i press `ctrl + enter`, this would be helpful when user want to trigger a send event, without leave the keyboard

**Comment by trueadm** [comment, marked as answer]
You can listen to the `KEY_ENTER_COMMAND`, which gets passed the `event` as the payload. `editor.addComment(KEY_ENTER_COMMAND, (event) => {…}, Priority)`

**Reply by guicoelho** [reply]
I think you mean `editor.registerCommand(KEY_ENTER_COMMAND, (event) => {…}, Priority)` 

Based on the docs I guess you wrote :) https://lexical.dev/docs/concepts/commands#editorregistercommand

**Reply by trueadm** [reply]
Yeah sorry for the typo there, I was on mobile when replying :)

**Comment by saumya66** [comment]
> For example, i want to add a shortcut when i press `ctrl + enter`, this would be helpful when user want to trigger a send event, without leave the keyboard

@IWANABETHATGUY did you make it happen? i need to implement the same too...🙂

**Reply by guicoelho** [reply]
You can use `editor.addComment(KEY_ENTER_COMMAND, (event) => {…}, Priority)` as @trueadm suggested.

If you want to check whether the `ctrl` key is also pressed, you can use `event.ctrlKey` in the handler. Like:

```
editor.registerCommand(
      KEY_ENTER_COMMAND,
      (event) => {
        if (event.ctrlKey) {
          // do something here
        }
      },
      Priority
    )
```
